squash/dashboard/views.py

```diff
@@ -177,10 +180,15 @@ def embed_bokeh(request, bokeh_app):
     bokeh_script = autoload_server(None, app_path="/{}".format(bokeh_app),
                                    url=bokeh_url)
 
+    template = loader.get_template('dashboard/embed_bokeh.html')
+
     context = {'bokeh_script': bokeh_script,
                'bokeh_app': bokeh_app}
 
-    return render(request, "dashboard/embed_bokeh.html", context)
+    response = HttpResponse(template.render(context, request))
+    response.set_cookie('django_full_path', request.get_full_path())
```

Comment:
I'm not sure if this will work in the production environment as the cookie would probably be readable by `squash.lsst.codes` but not `bokeh.lsst.codes`.  In am setting a domain of `lsst.codes` for the oauth proxies in front of each service.  This should be configurable so we don't have to hunt down a hard coded value in the future.

https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.HttpResponse.set_cookie